core/controllers/feedback_test.py

```diff
@@ -336,3 +339,109 @@ def _get_email(n):
             self.assertEqual(
                 response_dict['messages'][num + 1]['text'],
                 'New Message %s' % num)
+
+    def _return_true(self, thread_id, exploration_id):
+        return True
+
+    def _return_null(self, param_one, param_two, param_three, param_four):
+        return None
+
+    def test_create_fetch_and_action_suggestions(self):
+        # Create suggestions.
+        self.login(self.EDITOR_EMAIL)
+        response = self.testapp.get('/create/%s' % self.EXP_ID)
+        csrf_token = self.get_csrf_token_from_response(response)
+        # Create suggestions.
+        self.post_json(
+            '%s/%s' % (feconf.SUGGESTION_URL_PREFIX, self.EXP_ID), {
+                'exploration_version': 2,
+                'state_name': u'state_one',
+                'suggestion_content': {'old_content': {}},
+            }, csrf_token)
+        self.post_json(
+            '%s/%s' % (feconf.SUGGESTION_URL_PREFIX, self.EXP_ID), {
+                'exploration_version': 2,
+                'state_name': u'state_one',
+                'suggestion_content': {'old_content': {}},
+            }, csrf_token)
+        self.post_json(
+            '%s/%s' % (feconf.SUGGESTION_URL_PREFIX, self.EXP_ID), {
+                'exploration_version': 2,
+                'state_name': u'state_one',
+                'suggestion_content': {'old_content': {}},
+            }, csrf_token)
+        self.post_json(
+            '%s/%s' % (feconf.SUGGESTION_URL_PREFIX, self.EXP_ID), {
+                'exploration_version': 2,
+                'state_name': u'state_one',
+                'suggestion_content': {'old_content': {}},
+            }, csrf_token)
+        self.post_json(
+            '%s/%s' % (feconf.SUGGESTION_URL_PREFIX, self.EXP_ID), {
+                'exploration_version': 2,
+                'state_name': u'state_one',
+                'suggestion_content': {'old_content': {}},
+            }, csrf_token)
+        self.post_json(
+            '%s/%s' % (feconf.SUGGESTION_URL_PREFIX, self.EXP_ID), {
+                'exploration_version': 2,
+                'state_name': u'state_one',
+                'suggestion_content': {'old_content': {}},
+            }, csrf_token)
+
+        # Create a thread without suggestions.
+        self.post_json( 
+            '%s/%s' % (feconf.FEEDBACK_THREADLIST_URL_PREFIX, self.EXP_ID), {
+                'state_name': None,
+                'subject': u'New Thread ¡unicode!',
+                'text': 'Message 0'}, csrf_token)     
+
+        # Get a list of open threads without suggestions. 
+        response_dict = self.get_json( 
+            '%s/%s/%s/%s' % ( 
+                feconf.SUGGESTION_LIST_URL_PREFIX, self.EXP_ID, 'OPEN', False))  
```

Comment:
'open' should be lowercase and might as well go within the string pattern: %s/%s/open/false

Having said that, I recommend instead using a URL of the form %s/%s?status=open&has_suggestion=false